src/transformers/modeling_tf_t5.py

```diff
@@ -383,21 +383,14 @@ def _prune_heads(self, heads_to_prune):
 
     def call(
         self,
-        inputs,
+        hidden_states,
```

Comment:
No, I don't think so, because the PR is not touching `modeling_tf_t5.py` at all anymore (recall that I reverted the changes in that file in 6fe1cc08743ee8f8b76c622e5a0742413048e857).

I left out `TFT5MainLayer` because, although it fits the _naming pattern_ of other main layers in the project, it doesn't fit the _usage pattern_ (i.e. that the main layer accepts the model's inputs directly). The `test_keras_save_load` test implementation assumes that usage pattern, being based on `self.model_tester.prepare_config_and_inputs_for_common` ... so it fails on the T5 model.

Maybe the `@keras_serializable` decorator would work just fine for TFT5MainLayer, but verifying that would require a test case written more specifically for it, and I didn't sink my teeth into that.

I also left out three other `TF*MainLayer` classes simply because they failed the test and I couldn't quickly see why, see [comment above](https://github.com/huggingface/transformers/pull/3103#issuecomment-594213462)

Reply:
Ok make sense